### PR TITLE
fix workload fio iops

### PR DESCRIPTION
**Pull request by Shaoshian**
Signed-off-by: root <shianx.a.shao@intel.com>

### CODE REVIEW

**Comment by Shaoshian** [comment]
@xuechendi

File: analyzer/analyzer.py [hunk 1]
```diff
@@ -714,18 +714,33 @@ def process_fiolog_data(self, path, result):
...
+                    value = int(data[1])
+                    timestamp_sec = int(data[0])/time_shift
+                    if timestamp_sec > cur_sec:
+                        if cur_sec >= 0:
+                            self.tmp_res.append( self.iops_value )
+                            self.iops_value = 0
+                        cur_sec = timestamp_sec
+                    self.iops_value += value
+                if len(self.tmp_res) != 0:
+                    res.append(numpy.mean(self.tmp_res))
```
Comment (xuechendi) [inline analyzer/analyzer.py]: Why cal avg here? this function is aim to return a list iops per sec

**Comment by xuechendi** [comment]
@Shaoshian comment inline, result is not right. you only returned one avg number not a list.
Also, in commit, your git submit name is root, please change that to Shian.Shao

File: analyzer/analyzer.py (hunk 1 above)
Comment (Shaoshian) [inline analyzer/analyzer.py]: 不需要在这个函数里面求平均？  好 我去改一下。。

Comment (xuechendi) [inline analyzer/analyzer.py]: yes, please do a webui test before you submit the new fix, be aware that you should be able to see the iops line under workload tab, the second table

**Comment by Shaoshian** [comment]
@xuechendi   修改了，现在返回的是一个list了，我在web端跑了一个case试过了，没有问题，只是最后workload里面iops跟summary里面iops不太一样，我问过zhou,yuan了，他说误差小于1%就没事。。。

File: analyzer/analyzer.py [hunk 2]
```diff
@@ -714,18 +714,34 @@ def process_fiolog_data(self, path, result):
...
+                    timestamp_sec = int(data[0])/time_shift
+                    value = int(data[1])
+                    if timestamp_sec > cur_sec:
+                        if cur_sec >= 0:
+                            res.append(numpy.mean(self.tmp_res))
+                        cur_sec = timestamp_sec
+                    self.tmp_res.append( value )
+                if len(self.tmp_res) != 0:
+                    res.append(numpy.mean(self.tmp_res))
+        print result
```
Comment (xuechendi) [inline analyzer/analyzer.py]: please remove the line

File: analyzer/analyzer.py [hunk 3]
```diff
@@ -714,18 +714,34 @@ def process_fiolog_data(self, path, result):
...
+                    value = int(data[1])
+                    timestamp_sec = int(data[0])/time_shift
+                    if timestamp_sec > cur_sec:
+                        if cur_sec >= 0:
+                            self.tmp_res.append( self.iops_value )
+                            self.iops_value = 0
+                        cur_sec = timestamp_sec
+                    self.iops_value += value
+                if len(self.tmp_res) != 0:
+                    res.append(self.tmp_res)
```
Comment (xuechendi) [inline analyzer/analyzer.py]: why you append tmp_res to res？？

**Comment by xuechendi** [comment]
@Shaoshian , please remove the debug print codes. And seems the codes is still buggy, not showing the line chart, list data is printed as a string, did you tested on webui? And check the line chart?

**Comment by Shaoshian** [comment]
@xuechendi

**Pull request merged**